src/index.js

```diff
@@ -1,47 +1,64 @@
-import Mario from './arts/mario.ansi';
-import Minions from './arts/minions.ansi';
-import Parrot from './arts/parrot.ansi';
-import Ironman from './arts/ironman.ansi';
-import Panda from './arts/panda.ansi';
-import Link from './arts/link.ansi';
-import fs from 'fs';
-
-
-let Arts = [
-  { name: 'mario',   path: __dirname + Mario },
-  { name: 'minions', path: __dirname + Minions },
-  { name: 'parrot', path: __dirname + Parrot },
-  { name: 'ironman', path: __dirname + Ironman },
-  { name: 'panda', path: __dirname + Panda },
-  { name: 'link', path: __dirname + Link }
-]
-
-
-let getSpeechBubble = function(text) {
-  let lines = `+${'-'.repeat(text.length + 2)}+ \n| ${text} |\n+${'-'.repeat(text.length + 2)}+\n`
+const fs = require('fs');
+const arts = require('./arts');
```

Comment:
@ink8bit Sorry, but I'm really keen on es6 imports. Can you add them back? (here and everywhere)
https://github.com/airbnb/javascript#modules--use-them